lib/jolie-uri/src/main/java/com/damnhandy/uri/template/UriTemplateMatcherFactory.java

```diff
@@ -0,0 +1,11 @@
+package com.damnhandy.uri.template;
```

Comment:
If we do that, we can just remove this file.

Reply:
@fmontesi  we cannot do this because  

`UriTemplate,getReverseMatchPattern()` is a protected method which is why we have created this class in the same package `com.damnhandy.uri.template`

Reply:
@fmontesi can we close this